facade/errors.go

```diff
@@ -23,5 +23,5 @@ var ErrNilPeerState = errors.New("nil peer state")
 // ErrNilAccountState signals that a nil account state has been provided
 var ErrNilAccountState = errors.New("nil account state")
 
-// ErrNilTransactionSimulatorProcessor signals that a nil transaction simulator processor has been provided
+// ErrErrNilTransactionSimulatorProcessor signals that a nil transaction simulator processor has been provided
```

Comment:
Remove double ErrErr from comment and error variable also

Reply:
done